scripts/lc-builds/blueos_xl_omptarget.sh

```diff
@@ -37,8 +37,9 @@ cmake \
   -DBLT_CXX_STD=c++14 \
   -C ${RAJA_HOSTCONFIG} \
   -DENABLE_OPENMP=On \
-  -DENABLE_TARGET_OPENMP=On \
-  -DOpenMP_CXX_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
+  -DRAJA_ENABLE_TARGET_OPENMP=On \
+  -DBLT_OPENMP_COMPILE_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
+  -DBLT_OPENMP_LINK_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
   -DCMAKE_INSTALL_PREFIX=../install_${BUILD_SUFFIX} \
```

Comment:
Probably a bit pedantic but should these build script changes be in a separate PR?

Reply:
That would be best. I'm trying to reduce CI throughput issues and the script changes don't affect CI testing.